jdi-light/src/main/java/com/epam/jdi/light/actions/ActionProcessor.java

```diff
@@ -47,10 +47,22 @@
 @Aspect
 public class ActionProcessor {
 
+    public static MapArray<String, JFunc1<Object, Boolean>> CONDITIONS = map(
+            $("", result -> true),
+            $("true", result -> result instanceof Boolean && (Boolean) result),
+            $("false", result -> result instanceof Boolean && !(Boolean) result),
+            $("not empty", result -> result instanceof List && ((List) result).size() > 0),
+            $("empty", result -> result instanceof List && ((List) result).size() == 0)
+    );
+
     @Pointcut("execution(* *(..)) && @annotation(com.epam.jdi.light.common.JDIAction)")
-    protected void jdiPointcut() { }
+    protected void jdiPointcut() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Document empty method body](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4525538)

Reply:
fix please warnings (for example add `//nothing here` , I think it could be enough)

Reply:
done: 
    @Pointcut("execution(* *(..)) && @annotation(com.epam.jdi.light.common.JDIAction)")
    protected void jdiPointcut() { // empty
    }